examples/network_traffic_prediction/AirTrafficPrediction.ipynb

```diff
@@ -0,0 +1,735 @@
+   "id": "4f271746-c51e-4138-8543-b69b8fbc34bd",
+   "metadata": {},
+   "source": [
+    "GraphStorm provides a set of `Inferrer`s that can perform highly efficient inference for very large graphs. GraphStorm's `GSgnnNodePredictionInferrer` cannot handle the sliding window mehtod either. Similarly we extends the `GSgnnNodePredictionInferrer`, and specifically implements the autoregressive sliding window method by replacing true time serie values with the predicted ones, and then call the same customized `node_mini_batch_gnn_predict()` method during inference. Users can find the details in the `nr_models.py` file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "id": "270dc5b7-8cb0-4515-abc1-5c2bbca36836",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nr_models import NodePredictionInferrer4TS\n",
+    "\n",
+    "# Run inference on the inference dataset\n",
+    "infer = NodePredictionInferrer4TS(model)\n",
+    "\n",
+    "infer.infer(infer_dataloader,\n",
+    "            use_mini_batch_infer=True,\n",
+    "            save_embed_path=None,\n",
+    "            init_step=0,\n",
+    "            infer_steps=7,\n",
+    "            node_id_mapping_file='./gs_1p/node_mapping.pt',\n",
+    "            save_prediction_path='./predictions')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f252cfb9-ac19-463d-86e0-ba24347ab123",
+   "metadata": {},
+   "source": [
+    "Now let's check the prediction results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "id": "90af92cd-dfcb-4ae7-b29d-fe536e1f6ed3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "total 24K\n",
+      "-rw-rw-r-- 1 ubuntu ubuntu  15K Nov 21 00:06 predict-00000.pt\n",
+      "-rw-rw-r-- 1 ubuntu ubuntu 4.9K Nov 21 00:06 predict_nids-00000.pt\n"
+     ]
+    }
+   ],
+   "source": [
+    "# The GNN embeddings and predictions on the inference graph are saved to the folder named after the target_ntype\n",
+    "!ls -lh ./predictions/airport/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "733415bc-640d-4a68-8013-a6015d34f7a4",
+   "metadata": {},
+   "source": [
+    "#### Prediction results exploration\n",
+    "\n",
+    "The prediction results are two Pytorch tensor files. One is the actual prediction values for all airports and days defined by the `inti_step` and `infer_steps` arguments used in the `infer()` function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "id": "7fa9283a-f450-4a70-890b-bae33f2f7e2e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch as th\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cdff9ba4-4b6b-4615-a15e-3c68bbc92759",
+   "metadata": {},
+   "source": [
+    "The predictions will be $N \\time 7$, where $7$ is defined by the `infer_steps`. These values are actually the values from step 8 to 15."
```

Comment:
Not sure what you are talking about.
Do you mean, we start from init_step=0 and use 7 steps (dates) data to infer the 8th step? And we will infer 7 times?
Can you elaborate more about what we are predicting?

Reply:
Reorg the Result Explanation section to elaborate more.